timepiece/entries/forms.py

```diff
@@ -4,15 +4,21 @@
 from django import forms
 from django.db.models import Q
 
+from selectable import forms as selectable
 
 from timepiece import utils
 from timepiece.crm.models import Project, ProjectRelationship
 from timepiece.entries.models import Entry, Location, ProjectHours
+from timepiece.entries.lookups import ActivityLookup
 from timepiece.forms import (
     INPUT_FORMATS, TimepieceSplitDateTimeField, TimepieceDateInput)
 
 
 class ClockInForm(forms.ModelForm):
+    activity = selectable.AutoCompleteSelectField(
```

Comment:
This should be an `AutoComboboxSelectWidget`.  You can configure the widget in the class `Meta`:

```
        widgets = {
            'activity': selectable.AutoComboboxSelectWidget(lookup_class=ActivityLookup),
        }
```